### ISSUE TITLE
Viewport different with builtin dev server on different browsers

### ISSUE THREAD

**Issue by ssijak** [vuejs-templates/webpack#926]
I have just created demo app with webpack template importing this bootstrap 4 https://v4-alpha.getbootstrap.com/examples/narrow-jumbotron/ example into Hello.vue compontent.
I also added  `<meta content="width=device-width, initial-scale=1.00" name="viewport" />`

Problem is that with dev server, my viewport width is reported as 2100 in Chrome, and 1680 in Safari. And if I just build the project and run http-server from dist folder viewport width is 1680 in both browsers. Lost few hours on this but can't figure it out why it is like that.

Relevant recording is here : http://recordit.co/75Nuz5xXti

**Comment by ssijak** [comment]
OK, I figured it out. In `dev-server.js` server is started with this url `var uri = 'http://localhost:' + port`

For some reason, if you open a page in chrome (and only in Chrome, Firefox and Safari work fine) starting with localhost it ignores viewport meta tag and just slaps some viewport size but if you change the line to `var uri = 'http://127.0.0.1:' + port` it works as expected. 

Should I put a pull request for it to change the line?

**Comment by grakic** [comment]
I would check what Chrome extensions do you have enabled. I beleive some is interacting with the page content when it is loaded as localhost. Try testing it with all Chrome extensions disabled.

**Comment by LinusBorg** [comment]
@ssijak Did you find out anything more about this? I won't change the host to 127.0.0.1 just because it magically fixes this issue - if this is a chrome bug, we should document it.

And if it's just a weird side-effect of one extension you have running, that would be even better and we can close this.

**Comment by ssijak** [comment]
@LinusBorg While testing this I did disable all plugins which did not help. 
I could not connect this to anything but to the problem with chrome, it just slaps different viewport size when the page served by the webserver is opened with localhost in the url. Mind that the CSS and everything was loaded in both cases, only difference was that the chrome set larger viewport for localhost, and I tried all kinds of meta tags for viewport which changed nothing. For example : `<meta content="width=device-width, initial-scale=1.00" name="viewport" />`
